### PR TITLE
[ONEM-16231] Fix prebuildcleanup returning empty attributes as null

### PR DESCRIPTION
**Pull request by erenatas**
Updated buildwrappers.py and added a new test case for validation.

### CODE REVIEW

File: jenkins_job_wrecker/modules/buildwrappers.py [hunk 1]
```diff
@@ -143,26 +143,27 @@ def prebuildcleanup(top, parent):
...
-        elif element.tag == 'cleanupParameter':
-            preclean['check-parameter'] = element.text
-        elif element.tag == 'externalDelete':
-            preclean['external-deletion-command'] = element.text
-        elif element.tag == 'disableDeferredWipeout':
-            preclean['disable-deferred-wipeout'] = (element.text == 'true')
-        else:
-            raise NotImplementedError("cannot handle "
-                                      "XML %s" % element.tag)
+        if element.text is not None:
```
Comment (romanek-adam) [inline jenkins_job_wrecker/modules/buildwrappers.py]: You could:

    if element.text is None:
        continue

and leave the rest untouched?

Comment (erenatas) [reply jenkins_job_wrecker/modules/buildwrappers.py]: Done. Thanks.

**Review by vdanaila — APPROVED** [review]
Tested locally for CI/jobs/GitHub/jobs/verify-maintainers_onemw-samsung-selene/config.xml
Looks good.

File: tests/test_modules_buildwrappers.py [hunk 2]
```diff
@@ -21,3 +21,18 @@ def test_basic(self):
...
+        root = get_xml_root(filename=filename)
+        assert root is not None
+        parent = []
+        prebuildcleanup(root, parent)
+        assert len(parent) == 1
+        assert 'workspace-cleanup' in parent[0]
+        jjb_ws_cleanup = parent[0]['workspace-cleanup']
+        assert jjb_ws_cleanup['dirmatch'] is True
+        assert jjb_ws_cleanup['disable-deferred-wipeout'] is True
+        assert 'DO_WS_CLEANUP' not in jjb_ws_cleanup
```
Comment (romanek-adam) [inline tests/test_modules_buildwrappers.py]: this is just a value, not a key, so this assert will always be True, even without the fix

Comment (erenatas) [reply tests/test_modules_buildwrappers.py]: Yes, miswritten that. Thanks for noticing it.

**Review by romanek-adam — APPROVED** [review]

**Comment by romanek-adam** [comment]
@erenatas should this PR be upstreamed?

**Comment by erenatas** [comment]
> @erenatas should this PR be upstreamed?

Yes

**Pull request merged**